app/models/miq_widget.rb

```diff
@@ -513,11 +517,12 @@ def sync_schedule(schedule_info)
       raise _("Unsupported interval '%{interval}'") % {:interval => interval}
     end
 
-    sched = MiqSchedule.create!(
+    sched = existing_scheduler
+    sched ||= MiqSchedule.create!(
       :name          => description,
       :description   => description,
       :sched_action  => {:method => "generate_widget"},
-      :filter        => MiqExpression.new("=" => {"field" => "MiqWidget-id", "value" => id}),
```

Comment:
did you want to change this from a hash to a string?

Reply:
it was Hash in the first commit, second commit changed it